packages/core/src/middleware/interfaces/helmet.interface.ts

```diff
@@ -0,0 +1,249 @@
+/// <reference types="node" />
```

Comment:
You can remove the directive as the project tsconfig contains node in types already 
```json
"types": ["node", "reflect-metadata", "vitest/globals"]
```